README.md

```diff
@@ -55,6 +57,20 @@ And, once bundled, you can run a local server to use Conduit (defaults to [port
 ```sh
 npm run serve
 ```
+## Dev mode
+
+To run it in dev mode where saving your changes rebuilds and reloads the app, you can run the command below (which calls `spago build --watch`) 
+
+```sh
+npm run watch
+```
```

Comment:
This step should be unnecessary if using an IDE, such as vscode+PS extension, since it already recompiles upon save.